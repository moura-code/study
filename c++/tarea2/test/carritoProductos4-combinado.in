crearCarritoProductos
crearFecha 1/1/2001
crearProducto 1 Sillon 3
agregarCarritoProductos
crearFecha 3/2/2004
crearProducto 3 Mesa 15
agregarCarritoProductos
crearFecha 1/1/2005
crearProducto 5 Computadora 17
agregarCarritoProductos
crearFecha 5/4/2006
crearProducto 7 Pizza 23
agregarCarritoProductos
crearFecha 5/4/2007
crearProducto 4 Silla 12
agregarCarritoProductos
crearFecha 5/4/2008
crearProducto 6 Tele 23
agregarCarritoProductos
removerDeCarritoProductos 5
removerDeCarritoProductos 1
esVacioCarritoProductos
crearFecha 1/1/2005
crearProducto 5 Computadora 32
agregarCarritoProductos
obtenerProductoCarritoProductos 5
obtenerProductoCarritoProductos 4
obtenerProductoCarritoProductos 6
esVacioCarritoProductos
imprimirCarritoProductos
liberarCarritoProductos
Fin
