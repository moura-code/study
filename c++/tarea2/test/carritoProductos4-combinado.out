Bienvenido al programa principal de Mercado FINGer. Por favor ingrese su comando:
1> Carrito de productos creado de forma exitosa.
2> Fecha creada en forma exitosa.
3> Producto creado en forma exitosa.
4> Producto agregado de forma exitosa.
5> Fecha creada en forma exitosa.
6> Producto creado en forma exitosa.
7> Producto agregado de forma exitosa.
8> Fecha creada en forma exitosa.
9> Producto creado en forma exitosa.
10> Producto agregado de forma exitosa.
11> Fecha creada en forma exitosa.
12> Producto creado en forma exitosa.
13> Producto agregado de forma exitosa.
14> Fecha creada en forma exitosa.
15> Producto creado en forma exitosa.
16> Producto agregado de forma exitosa.
17> Fecha creada en forma exitosa.
18> Producto creado en forma exitosa.
19> Producto agregado de forma exitosa.
20> Producto con id 5 removido con exito.
21> Producto con id 1 removido con exito.
22> El carrito de productos NO es vacio.
23> Fecha creada en forma exitosa.
24> Producto creado en forma exitosa.
25> Producto agregado de forma exitosa.
26> Producto: 5
Computadora
Precio: 32
Ingresado el: 1/1/2005
27> Producto: 4
Silla
Precio: 12
Ingresado el: 5/4/2007
28> Producto: 6
Tele
Precio: 23
Ingresado el: 5/4/2008
29> El carrito de productos NO es vacio.
30> Producto: 3
Mesa
Precio: 15
Ingresado el: 3/2/2004
Producto: 4
Silla
Precio: 12
Ingresado el: 5/4/2007
Producto: 5
Computadora
Precio: 32
Ingresado el: 1/1/2005
Producto: 6
Tele
Precio: 23
Ingresado el: 5/4/2008
Producto: 7
Pizza
Precio: 23
Ingresado el: 5/4/2006
31> Carrito de productos liberado con exito.
32> Fin.
